rosdep/base.yaml

```
@@ -2865,6 +2865,17 @@ libedit-dev:
   nixos: [libedit]
   rhel: [libedit-devel]
   ubuntu: [libedit-dev]
+e2fsprogs:
+  alpine: [e2fsprogs]
+  arch: [e2fsprogs]
+  debian: [e2fsprogs]
+  fedora: [e2fsprogs]
+  gentoo: [sys-libs/e2fsprogs-libs]
+  macports: [e2fsprogs]
+  nixos: [e2fsprogs]
+  opensuse: [e2fsprogs-libs]
+  rhel: [e2fsprogs-devel]
```

This provides the headers, but I think you want the utilities: http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/e2fsprogs-1.45.6-1.el8.x86_64.rpm
```suggestion
  rhel: [e2fsprogs]
```